From 6e1970f0ff5ffbddae54403554ab091c455992ed Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Mon, 30 Mar 2020 23:56:20 +0800
Subject: [PATCH 07/22] driver: gpu: drm: correct the path of head files

It is need to use the full path of head files, so add $(srctree)
before the relative path to generate a full path.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dcss/Makefile  | 2 +-
 drivers/gpu/drm/imx/dpu/Makefile   | 2 +-
 drivers/gpu/drm/imx/lcdif/Makefile | 2 +-
 drivers/gpu/imx/dpu-blit/Makefile  | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/imx/dcss/Makefile b/drivers/gpu/drm/imx/dcss/Makefile
index f7f7e9e9b550..0c8c8c148e9f 100644
--- a/drivers/gpu/drm/imx/dcss/Makefile
+++ b/drivers/gpu/drm/imx/dcss/Makefile
@@ -1,4 +1,4 @@
-ccflags-y += -Idrivers/gpu/drm/imx
+ccflags-y += -I$(srctree)/drivers/gpu/drm/imx
 
 imx-dcss-crtc-objs := dcss-crtc.o dcss-plane.o dcss-kms.o
 obj-$(CONFIG_DRM_IMX_DCSS) += imx-dcss-crtc.o
diff --git a/drivers/gpu/drm/imx/dpu/Makefile b/drivers/gpu/drm/imx/dpu/Makefile
index b938c97922a7..e40faea217d5 100644
--- a/drivers/gpu/drm/imx/dpu/Makefile
+++ b/drivers/gpu/drm/imx/dpu/Makefile
@@ -1,4 +1,4 @@
-ccflags-y += -Idrivers/gpu/drm/imx
+ccflags-y += -I$(srctree)/drivers/gpu/drm/imx
 
 imx-dpu-crtc-objs := dpu-crtc.o dpu-plane.o dpu-kms.o
 imx-dpu-crtc-$(CONFIG_DEBUG_FS) := dpu-crc.o
diff --git a/drivers/gpu/drm/imx/lcdif/Makefile b/drivers/gpu/drm/imx/lcdif/Makefile
index fcdecff9e861..5b2634e5bc54 100644
--- a/drivers/gpu/drm/imx/lcdif/Makefile
+++ b/drivers/gpu/drm/imx/lcdif/Makefile
@@ -1,4 +1,4 @@
-ccflags-y += -Idrivers/gpu/drm/imx
+ccflags-y += -I$(srctree)/drivers/gpu/drm/imx
 
 imx-lcdif-crtc-objs := lcdif-crtc.o lcdif-plane.o lcdif-kms.o
 obj-$(CONFIG_DRM_IMX_LCDIF) += imx-lcdif-crtc.o
diff --git a/drivers/gpu/imx/dpu-blit/Makefile b/drivers/gpu/imx/dpu-blit/Makefile
index a1c760d36ad8..85aa6087a3ad 100644
--- a/drivers/gpu/imx/dpu-blit/Makefile
+++ b/drivers/gpu/imx/dpu-blit/Makefile
@@ -1,4 +1,4 @@
-ccflags-y += -Idrivers/gpu/imx/dpu
+ccflags-y += -I$(srctree)/drivers/gpu/imx/dpu
 
 imx-dpu-blit-objs := dpu-blit.o
 
-- 
2.17.1

